services/clusters-keeper/src/simcore_service_clusters_keeper/modules/dask.py

```diff
@@ -49,42 +19,34 @@ async def _wrap_client_async_routine(
     return await client_coroutine
 
 
-async def is_gateway_busy(*, url: AnyUrl, gateway_auth: SimpleAuthentication) -> bool:
-    basic_auth = dask_gateway.BasicAuth(
-        username=gateway_auth.username,
-        password=gateway_auth.password.get_secret_value(),
-    )
-    async with dask_gateway.Gateway(
-        address=f"{url}",
-        auth=basic_auth,
-        asynchronous=True,
-    ) as gateway:
-        cluster_reports = await asyncio.wait_for(gateway.list_clusters(), timeout=5)
-        if not cluster_reports:
-            _logger.info("no cluster in gateway, nothing going on")
-            return False
-        assert len(cluster_reports) == 1  # nosec
-        async with gateway.connect(
-            cluster_reports[0].name, shutdown_on_close=False
-        ) as dask_cluster, dask_cluster.get_client() as client:
-            datasets_on_scheduler = await _wrap_client_async_routine(
-                client.list_datasets()
-            )
+async def ping_scheduler(url: AnyUrl) -> bool:
```

Comment:
THOUGHT: at some point i would like to distinguish health-checks at the transport level and at the application level. I assume ping is the former? i.e. the container is reachable but the app still not fully initialized? Do we really need both?

Reply:
so here the ping_scheduler tries to connect with the dask-scheduler.
Even chatty just proposed the same name... let me know what you prefer